modules/@apostrophecms/schema/ui/apos/components/AposSchema.vue

```diff
@@ -101,9 +101,12 @@ export default {
         // The doc might be swapped out completely in cases such as the media
         // library editor. Repopulate the fields if that happens.
         if (
-          this.fieldState._id &&
-          (newVal.data._id !== this.fieldState._id.data)
```

Comment:
This is not a field,  it concerns me that we're treating it as one. But, taht wasn't first introduced here.